xml/hardware_nvdimm.xml

```diff
@@ -220,22 +214,20 @@
      </para>
      <para>
       Apart from <literal>devdax</literal> namespaces, all other types must be
-      formatted with a file system, just as with a conventional drive.
-      &productname; supports the <literal>ext2</literal>,
-      <literal>ext4</literal> and <literal>XFS</literal> file systems for this.
+      formatted with a file system such as <literal>ext2</literal>,
+      <literal>ext4</literal> or <literal>XFS</literal>, just as with a
+      conventional drive.
      </para>
-     <note>
-      <para>
-       The <literal>ext2</literal> file system does not support DAX.
-      </para>
-     </note>
     </listitem>
    </varlistentry>
    <varlistentry>
     <term>Direct Access (DAX)</term>
     <listitem>
      <para>
-      Directly <literal>mmap()</literal> NV-RAM into a process' address space.
```

Comment:
Is ```<literal>mmap()</literal>``` used as verb here? If yes, it sounds like technical jargon to me (maybe I'm also lacking context here because I did not read the complete chapter). Could you perhaps also specify what   ```<literal>mmap()</literal>``` is? A function?

Reply:
A fair point. mmap() is a Unix system call. I have removed this reference and explained it in words instead.